### ISSUE TITLE
[error] [xiaomi-plug:socket] ReferenceError: miDevicesUtils is not defined

### ISSUE THREAD

**Issue by artemsuv** [pierrecle/node-red-contrib-mi-devices#33]
Hello @pierrecle 
I create flow from your example, and leave only this part:
![image](https://user-images.githubusercontent.com/3994398/35771307-36a5b4bc-0933-11e8-8c8c-a3c97eefc098.png)
When inject node send message to socket node, i recive next error:
```
03.02.2018, 22:41:59node: socket
msg : error
"ReferenceError: miDevicesUtils is not defined"
```

**Comment by artemsuv** [comment]
Fix #34

**Comment by bemble** [comment]
Thanks for the fix ;)
